code/modules/mob/living/carbon/xenomorph/castes/king/castedatum_king.dm

```diff
@@ -18,14 +18,14 @@
 	speed = 0
 
 	// *** Plasma *** //
-	plasma_max = 900
-	plasma_gain = 40
+	plasma_max = 1000 //Bumped up
+	plasma_gain = 80 //Bumped up
 
 	// *** Health *** //
-	max_health = 500
+	max_health = 400 //Bumped down to T3 crusher levels, though rises up
 
 	// *** Evolution *** //
-	upgrade_threshold = 350
+	upgrade_threshold = 250 //Bumped down so they're more useful
```

Comment:
It makes more sense to me to keep the evolution rate at around the same level as the queen's, both thematically and mechanically. Since it *is* a very strong caste that can be evolved into right from the get-go.
That is,
Young > Mature (350)
Mature > Elder (850)
Elder > Ancient (1850)

Reply:
Fair enough